build/scripts/tests.sh

```diff
@@ -45,8 +45,8 @@ do
     runtimeconfig_json="${file_name%.*}".runtimeconfig.json
 
     # If the user specifies a test on the command line, only run that one
-    # "${2:-}" => take second arg, empty string if unset (regex always matches if empty)
-    if [[ ! "${file_name}" =~ "${2:-}" ]]
+    # "${2:-}" => take second arg, empty string if unset
+    if [[ ("${2:-}" != "") && (! "${file_name}" =~ "${2:-}") ]]
```

Comment:
This is a disturbing OS difference, but LGTM.

Reply:
Agree. Tried to lookup docs on this (mostly to verify I wasn't interpreting this wrong). But experimentally at least it appears to be a difference between the OS.